_security-plugin/multi-tenancy/mt-agg-view.md

```diff
@@ -0,0 +1,74 @@
+{: .warning}
+
+Aggregate view for saved objects allows a user who has access to multiple tenants to see all saved objects associated with those tenants in a single view without having to switch between tenants to do so. This includes both tenants created by the user and tenants shared with the user. Aggregate view introduces a Tenants column in the Saved Objects table that gives the user the option to filter by tenants and make visible their associated saved objects.
+
+Once a user identifies a saved object of interest, the user can then switch to that tenant to work with the object.
+
+To access saved objects, expand the top menu and select **Management > Stack Management > Saved Objects**. The Saved Objects window opens. By default, all tenants the user has permissions to are dispalayed along with all saved objects associated with the tenants.
+
+<img src="{{site.url}}{{site.baseurl}}/images/Security/agg-view-saved-objects.png" alt="Dashboards Saved Objects view with tenant object aggregation" width="700">
+
+As an experimental feature, Aggregate view of saved objects is kept behind a feature flag and must be enabled in the opensearch_dashboards.yml before the feature is made available. See [Enabling aggregate view](#enabling-aggregate-view-for-saved-objects) below for more on this.
+{: .note }
+
+### Feature benefits
+
+- Implementing an aggregate view of all saved objects on one screen allows a user to quickly locate an object of interest and determine which tenant is associated with it. Once the user locates an object, the user can select the appropriate tenant and work with the object.
+- This feature also adds a Tenants column to the Saved Objects table, which allows users to filter the view by tenants and their associated saved objects.
+
+### Plans for future development
+
+In subsequent releases, we plan to expand on the functionality of this feature to include the ability to perform actions directly from aggregate view and share items without having to first select a specific tenant. In the longer term, OpenSearch plans to evolve multi-tenancy so that it becomes a much more flexible tool for sharing objects among users and easier to do so by a more sophisticated way of assigning the roles and permissions that faciliate sharing. To learn more about the features being proposed for future releases, see the GitHub issue [Dashboards object sharing](https://github.com/opensearch-project/security/issues/1869).
```

Comment:
```suggestion
In subsequent releases, we plan to expand the functionality of this feature to include the ability to perform actions directly from aggregate view and share items without having to first select a specific tenant. In the longer term, OpenSearch plans to evolve multi-tenancy so that it becomes a much more flexible tool for sharing objects among users and uses a more sophisticated way of assigning the roles and permissions that facilitate sharing. To learn more about the features being proposed for future releases, see the GitHub issue [Dashboards object sharing](https://github.com/opensearch-project/security/issues/1869).
```